src/Core/Scheduler/Cache.js

```diff
@@ -97,19 +123,31 @@ const Cache = {
      * @name module:Cache.delete
      * @function
      *
-     * @param {string} key
-     *
-     * @return {boolean} - Confirmation that the entry has been deleted.
+     * @param {string} key1
+     * @param {string} [key2]
+     * @param {string} [key3]
      */
-    delete: key => data.delete(key),
+    delete: (key1, key2, key3) => {
+        if (data[key1] == undefined) {
+            throw Error('Please specify at least a key of something to delete');
+        } else if (data[key1][key2] == undefined) {
+            delete data[key1];
+        } else if (data[key1][key2][key3] == undefined) {
+            delete data[key1][key2];
+        } else {
+            delete data[key1][key2][key3];
+        }
+    },
```

Comment:
Destroying only the `key1` could be useful to clean the cache of one `source/Layer`?

Reply:
It is the case: if `key2 == undefined`, you are in the case `data[key1][key2] == undefined`, as `data[key1][undefined] == undefined`. Deleting then `data[key1]` deletes all the children of `delete[key1]`.